site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -101,6 +101,8 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `agones.image.controller.name`                           | Image name for the controller                                                                                                                                                                                           | `agones-controller`                |
 | `agones.image.controller.pullPolicy`                     | Image pull policy for the controller                                                                                                                                                                                    | `IfNotPresent`                     |
 | `agones.image.controller.pullSecret`                     | Image pull secret for the controller, allocator, sdk and ping image. Should be created both in `agones-system` and `default` namespaces                                                                                 | \`\`                               |
+| `agones.image.extensions.name`                           | Image name for extensions                                                                                                                                                                                          | `agones-extensions`                |
```

Comment:
There's a handy block for new config options we keep at the bottom of the page, these can go in there until next release where they will get merged as part of the release process.